src/main/java/net/daporkchop/toobeetooteebot/util/cache/data/chunk/ChunkCache.java

```diff
@@ -78,7 +78,7 @@ public Column get(int x, int z) {
 
     public void remove(int x, int z) {
         CACHE_LOG.debug("Server telling us to uncache chunk (%d, %d)", x, z);
-        if (this.cache.remove(new Vec2i(x, z)) == null) {
+        if (this.cache.remove(new Vec2i(x, z)) == null && CONFIG.log.sendWarning) {
```

Comment:
this message should be being printed: the only things that should be able to cause it are be a bug in the server or a bug in the bot.